docs/sources/awscodecommit.md

```diff
@@ -1,119 +1,123 @@
-> called `awscreds`. This Secret can be generated with the following command:
->
-> ```console
-> $ kubectl -n <my_namespace> create secret generic awscreds \
->   --from-literal=aws_access_key_id=<my_key_id> \
->   --from-literal=aws_secret_access_key=<my_secret_key>
-> ```
->
-> Alternatively, credentials can be used as literal strings instead of references to Kubernetes Secrets by replacing
-> `valueFrom` attributes with `value` inside API objects' manifests.
+A fully qualified ARN is required to uniquely identify the AWS CodeCommit repository.
 
-### As a AWSCodeCommitSource object
-
-Copy the sample manifest from `config/samples/awscodecommitsource.yaml` and replace the pre-filled `spec` attributes
-with the values corresponding to your _AWS CodeCommit_ repository. Then, create that `AWSCodeCommitSource` object in
-your Kubernetes cluster:
+The easiest way to obtain the ARN of a CodeCommit repository is by using the [AWS CLI][aws-cli]. The following command
+retrieves the information of a repository called `triggermeshtest` in the `us-west-2` region:
 
 ```console
-$ kubectl -n <my_namespace> create -f my-awscodecommitsource.yaml
+$ aws codecommit get-repository --repository-name triggermeshtest --region us-west-2
+{
+    "repositoryMetadata": {
+        "accountId": "123456789012",
+        "repositoryId": "510acd3d-b96d-473c-bbe4-a8c6799d02a9",
+        "repositoryName": "triggermeshtest",
+        "defaultBranch": "main",
+        "lastModifiedDate": "2020-07-20T20:54:27.806000+02:00",
+        "creationDate": "2020-07-20T20:49:12.324000+02:00",
+        "cloneUrlHttp": "https://git-codecommit.eu-central-1.amazonaws.com/v1/repos/triggermeshtest",
+        "cloneUrlSsh": "ssh://git-codecommit.eu-central-1.amazonaws.com/v1/repos/triggermeshtest",
+        "Arn": "arn:aws:codecommit:eu-central-1:123456789012:triggermeshtest"
+    }
+}
 ```
 
-### As a ContainerSource object
+If you don't have the AWS CLI installed on your workstation, you can use the template below to compose a fully
+qualified ARN of a CodeCommit repository.
 
-Copy the sample manifest from `config/samples/awscodecommit-containersource.yaml` and replace the pre-filled environment
-variables under `env` with the values corresponding to your _AWS CodeCommit_ repository. Then, create that
-`ContainerSource` object in your Kubernetes cluster:
+```
+arn:aws:codecommit:{awsRegion}:{awsAccountId}:{repositoryName}
+```
 
-```console
-$ kubectl -n <my_namespace> create -f my-awscodecommit-containersource.yaml
+### API credentials
+
+The TriggerMesh AWS CodeCommit event source authenticates calls to the AWS CodeCommit API using an [Access
+Key][accesskey]. The page at this link contains instructions to create an access key when signed either as the root user
+or as an IAM user. Take note of the **Access Key ID** and **Secret Access Key**, they will be used to create and
```

Comment:
to create "an" instance ?

Reply:
Nice catch

Reply:
Done in [bf97c766](https://github.com/triggermesh/docs/compare/80bed3913e9781b28750273e0892c2e40bdeb67e..bf97c7662a25053044023d85010f2ccfcad323a1)